### PR TITLE
fix bbt in  14day can openlink

### PR DESCRIPTION
**Pull request by CS6**
修正第14天時，顯示奶茶圖案並且可以點擊的功能

### CODE REVIEW

File: src/app/daily/daily-overview/daily-overview.component.html [hunk 1]
```diff
@@ -5,12 +5,16 @@
   /* border:1px solid #000; */
   ">
     <div class="TA-AE" style="height: 50vw;width: 50vw;margin: 10px; ">
-    <ng-lottie [options]="options" (animationCreated)="animationCreated($event)"
+    <ng-lottie  
+    (click)="goToLink('https://www.w3schools.com/html/')"
```
Comment (shc261392) [inline src/app/daily/daily-overview/daily-overview.component.html:9]: Not sure if `<a>` tag can be used along with `<ng-lottie>`?
If yes, maybe use `<a *ngIf="WhatIsItToday >= 14" href="...">` would be cleaner?

Comment (shc261392) [reply src/app/daily/daily-overview/daily-overview.component.html:9]: Also it seems to me that https://www.mydata.org.tw/ would be a more appropriate placeholder

Comment (CS6) [reply src/app/daily/daily-overview/daily-overview.component.html:9]: <a *ngIf="WhatIsItToday> <ng-lottie/><a>
觸發前 lottie 不顯示
 <ng-lottie><a/> </ng-lottie>
無法使用

Comment (CS6) [reply src/app/daily/daily-overview/daily-overview.component.html]: @shc261392 
已替換Link
移除 {{WhatIsItToday}} 
在裝置上測試過了

**Comment by shc261392** [comment]
LGTM

**Pull request merged**